pygotham/talks/forms.py

```diff
@@ -23,7 +23,12 @@ class TalkSubmissionForm(ModelForm):
 
     class Meta:
         model = Talk
-        exclude = ('status',)
+        exclude = ('abstract',
+                   'objectives',
+                   'outline',
+                   'status',
+                   'target_audience',
+                   'type')
```

Comment:
I'm not a fan of this stylistically for the same reasons I always prefer trailing commas, mostly related to commit diffs. That said, the functionality seems sound.